dwave/preprocessing/composites/spin_reversal_transform.py

```diff
@@ -25,15 +25,32 @@
 class SpinReversalTransformComposite(dimod.core.Sampler, dimod.core.Composite):
     """Composite for applying spin reversal transform preprocessing.
 
-    Spin reversal transforms (or "gauge transformations") are applied
-    by randomly flipping the spin of variables in the Ising problem. After
-    sampling the transformed Ising problem, the same bits are flipped in the
-    resulting sample [#km]_.
+    A spin-reversal transform does not alter the Ising problem; the 
+    transform simply amounts to reinterpreting spin up as spin down, 
+    and visa-versa, for a particular spin. The technique works as follows: 
+    Given an :math:`n`-variable Ising problem, we can select a random
+    :math:`g\in\{\pm1\}^n` and transform the problem via 
+    :math:`h_i\mapsto h_ig_i` and :math:`J_{ij}\mapsto J_{ij}g_ig_j`. 
+    A spin-reversal transform does not alter the mathematical nature 
+    of the Ising problem. Solutions :math:`s` of the original
```

Comment:
I don't know what "does not alter the mathematical nature of the Ising problem" means.